pkg/daemon/agent/flexvolume/server.go

```diff
@@ -35,67 +35,70 @@ import (
 const (
 	UnixSocketName           = ".rook.sock"
 	FlexvolumeVendor         = "rook.io"
-	FlexvolumeDriver         = "rook"
+	LegacyFlexDriverName     = "rook"
```

Comment:
I don't think Legacy is the right name we want to use anymore, because this will actually be the recommended one that gets used in the mainline scenarios through our docs, right?  The integration tests are the only ones that will use the namespaced driver name if I understand correctly.

Reply:
agreed, will rename it